### ISSUE TITLE
remove pupmessage

### ISSUE THREAD

**Issue by zigbert** [puppylinux-woof-CE/woof-CE#610]
I have worked through mavrothal's list of scripts including pupmessage.
These boxes are now pointed to /usr/lib/gtkdialog/box_ok
please remove /usr/bin/pupmessage

The list:
--------------------------------------------------------------------------------
for pupmessage is even worse:

[OK] /usr/local/bin/drive_all uses it
[OK] /usr/local/petget/service_pack.sh uses it
[OK] /usr/local/petget/petget uses it
[OK] /usr/local/petget/findnames.sh uses it
[OK] /usr/local/petget/removepreview.sh uses it
[OK] /usr/local/petget/devx_service_pack.sh uses it
[OK] /usr/local/simple_network_setup/sns uses it
[OK] /usr/local/video_upgrade/video_upgrade_wizard uses it
[OK] /usr/sbin/shutdownconfig uses it
[OK] /usr/sbin/xorgwizard uses it
[OK] /usr/sbin/remasterpup2 uses it
[OK] /usr/sbin/sfsget uses it
[OK] /usr/sbin/filemnt uses it
[OK] /usr/sbin/pmount uses it
[OK] /usr/sbin/puppyinstaller uses it
[OK] /usr/sbin/bcrypt_gui uses it
[OK] /usr/sbin/bootflash uses it
[OK] /usr/sbin/bootmanager uses it
[OK] /usr/sbin/burniso2cd uses it
[OK] /usr/sbin/cdburner-wizard uses it
[OK] /usr/sbin/loginmanager uses it
[OK] /usr/sbin/resizepfile.sh uses it
[OK] /usr/sbin/savesession-dvd uses it
[OK] /usr/sbin/user2root uses it
[OK] /usr/sbin/xdelta_gui uses it
[OK] /usr/local/petget/installpkg.sh uses it
/usr/bin/pupmessage uses it

**Comment by dimkr** [comment]
Is there anything in this list that is also supposed to work from the console, without X?

**Comment by zigbert** [comment]
pupmessage points to pupdialog which relies on gtk. pupmessage can't produce any message without a running X.

But yes, ie. xorgwizard is supposed to work _also_ without X, but in those cases it uses 'dialog' to produce the message box.

**Comment by mavrothal** [comment]
My list comes from a running "stock" puppy. I do not know if other common/noarch pets use it.
Would  a convention-pupmessage that will change the text field syntax to use box_ok work if we can guess correctly the icon (search for "error"/"sorry")? If not, what about popping up a message that is depreciated and to use box_ok instead?

**Comment by zigbert** [comment]
My very personal opinion!
- If we want to stay stable, let's stick to what we got.
- If we want to ship new features we have to accept a certain level of risk.
- If we want to have fun (my only Puppy-motivation), we simply need to evolve. To ensure evolving doesn't lead to chaos, we need to cleanup. Introducing of rootfs-packages is probably the biggest potential cleanup and a great platform for evolving.
- If we want to clean up, we need to remove code - not implement new; if this, if that... Cleanup should lead to simplicity. This is also my opinion on the mini-icons and your latest jwm-argue. I can't see the problem to state that Puppy doesn't include 16x16.xpm, doesn't support pupmessage or that jwm_config requires jwm-2.3.2

KISS

NB! I have checked my good old Slacko-5.7, and there is no other apps using pupmessage than those in mavrothal's list
